units/host_allocator.go

```diff
@@ -246,6 +247,33 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	hostQueueRatio := float32(timeToEmpty.Nanoseconds()) / float32(distroQueueInfo.MaxDurationThreshold.Nanoseconds())
 	noSpawnsRatio := float32(timeToEmptyNoSpawns.Nanoseconds()) / float32(distroQueueInfo.MaxDurationThreshold.Nanoseconds())
 
+	lowRatioThresh := float32(.25)
+	terminateExcess := distro.HostAllocatorSettings.HostsOverallocatedRule == evergreen.HostsOverallocatedTerminate
+	if terminateExcess && hostQueueRatio > 0 && hostQueueRatio < lowRatioThresh {
+
+		killableHosts := int(float32(len(upHosts)) * (1 - hostQueueRatio))
+		newCapTarget := len(upHosts) - killableHosts
+
+		if newCapTarget < distro.HostAllocatorSettings.MinimumHosts {
+			newCapTarget = distro.HostAllocatorSettings.MinimumHosts
+		}
+		lowCountFloor := 5
```

Comment:
can this also be a constant? And can you comment this a little bit? Did we choose this number for any particular reason?

Reply:
No, I completely just made it up based on a low-host-count distro I happened to be looking at at the time. It could easily be 3 or 7.